crates/relayer/src/chain/ckb4ibc.rs

```diff
@@ -561,6 +563,20 @@ impl ChainEndpoint for Ckb4IbcChain {
         &mut self,
         tracked_msgs: TrackedMsgs,
     ) -> Result<Vec<IbcEventWithHeight>, Error> {
+        // specificly manage Ckb4Ibc endpoint, because Axon's light client on Ckb is its metadata cell
+        if let TrackingId::Static("create client") = tracked_msgs.tracking_id() {
```

Comment:
Use a string constant?

Reply:
that's fine, but in the framework of Hermes, it uses literal string too, and this is just a quick process to pass our test